client/src/components/Verejne/Verejne.scss

```diff
@@ -0,0 +1,238 @@
+html,
+body {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+    background-color: #f1f4f5;
```

Comment:
Good point. I think we should move ALL colors to the specific file because it seems there are too many colors (there are like 6 shades of white...). I think when we list them all in a file we will have easier time refactoring and doing design changes.